pkg/operator/rbaccontroller/rbaccontroller.go

```diff
@@ -0,0 +1,99 @@
+package rbaccontroller
+
+import (
+	"context"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	listerrbacv1 "k8s.io/client-go/listers/rbac/v1"
+	"k8s.io/klog"
+)
+
+const (
+	// openShiftManagedRBACAnnotation is the name of an annotation which prevents reconciliation if set to "false"
```

Comment:
Doesn't it mean that the `openshiftapiserver.rbac.authorization.openshift.io/managed: true` means that it is owned by the operator, and therefore reconciled against the latest version? It will never be false. Or can and may a user set it to false?

Reply:
Yes. It is owned and reconciled only when `openshiftapiserver.rbac.authorization.openshift.io/managed: true`. If this annotation is missing or when user set it to false, reconcile is skipped.